test/e2e/anthill_test.go

```diff
@@ -0,0 +1,80 @@
+package e2e
+
+import (
+	goctx "context"
+	"testing"
+	"time"
+
+	framework "github.com/operator-framework/operator-sdk/pkg/test"
+	"github.com/operator-framework/operator-sdk/pkg/test/e2eutil"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	apis "github.com/gluster/anthill/pkg/apis"
```

Comment:
can we change the importing order here?
import `go pkg` , `antill pkg` and `third-party libs`

Reply:
done